content/500-support/100-help-articles/200-working-with-many-to-many-relations.mdx

```diff
@@ -0,0 +1,190 @@
+---
+title: 'Working with many-to-many relations with Prisma'
+metaTitle: 'Working with many-to-many relations with Prisma'
+metaDescription: 'Learn how you can model and query implicit and explicit many-to-many relations with Prisma'
+---
+
+## Problem
+
+Lots of users have queries about can one model many-to-many relations and query for the data using Prisma. Here are two examples one implicit and one explicit to show how you can handle these with Prisma.
```

Comment:
```suggestion
Modeling and querying many-to-many relations in relational databases can be challenging. This article shows two examples how this can be approached with Prisma. The first example uses an [implicit](../../concepts/components/prisma-schema/relations#implicit-many-to-many-relations) and the second one uses an [explicit](../../concepts/components/prisma-schema/relations#explicit-many-to-many-relations) many-to-many relation.
```

Reply:
Side-note: I suggest to not use the term "query" when talking about user requests, just because the terms is already super ambiguous/overloaded 😄 in most cases, I think "question" feels a lot more appropriate.